.cicd/pipeline.yml

```diff
@@ -211,6 +211,14 @@ steps:
     timeout: 10
     skip: ${SKIP_MOJAVE}${SKIP_PACKAGE_BUILDER}
 
+  - wait
+  
+  - label: ":docker::ubuntu: Docker - Build 18.04 Docker Image"
+    command: "./.cicd/create-docker-from-binary.sh"
+    agents:
+      queue: "automation-eks-eos-builder-fleet"
```

Comment:
The `automation-eks-eos-builder-fleet` is a beefier fleet than what we need to run aptitude. Let's switch to the `automation-eks-eos-tester-fleet`, which is a lighter instance type.

Reply:
Done.